private_gpt/components/ingest/readers/rdfreader.py

```diff
@@ -0,0 +1,92 @@
+# mypy: ignore-errors
+
+"""Read RDF files.
+
+This module is used to read RDF files.
+It was created by llama-hub but it has not been ported
```

Comment:
So, it was ported to llama-index 0.1.0 with fixes, right? This sentence is a little bit confusing...